### PR TITLE
create and style the LongTextInput component

### PR DESCRIPTION
**Pull request by demirsergen**

### CODE REVIEW

**Review by daleinen7 — CHANGES_REQUESTED** [review]
Can you add this component to the "components" page so that we can test it out?

File: src/components/LongTextInput.jsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  }
+
+  span {
+    display: block;
+    margin-left: auto;
+    font-weight: 400;
+    color: var(--dark-grey);
+  }
+
+  .max-word {
```
Comment (daleinen7) [inline src/components/LongTextInput.jsx:34]: Looks like there shoudl also be an outline around the whole component when there is an error. 
Screen Shot 2023-01-28 at 5 00 02 PM

File: src/components/LongTextInput.jsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    color: var(--dark-red);
+  }
+`
+
+const LongTextInput = () => {
+  const [text, setText] = useState('')
+  const [wordCount, setWordCount] = useState(0)
+  const [maxWordCount, setMaxWordCount] = useState(300)
```
Comment (daleinen7) [inline src/components/LongTextInput.jsx:42]: Let's get this value from props but set a default to "300".

**Review by daleinen7 — CHANGES_REQUESTED** [review]
Looks good so far. I have a few minor style requests, and then I want to confirm with Joseph something before we merge this. 

Let's match the active and focus states of the text inputs. Rather than having the outline be around the text, let's have it around the whole component. 
Screen Shot 2023-01-28 at 4 51 21 PM
Screen Shot 2023-01-28 at 4 52 07 PM

**Comment by demirsergen** [comment]
Made the requested styling changes. Once we have an answer whether it's word or character count, I can fix that as well.

**Comment by daleinen7** [comment]
> https://deploy-preview-27--codeforchicago.netlify.app

Yeah looks good otherwise. I'll let you know what I hear from Joseph.

**Review by daleinen7 — APPROVED** [review]

**Pull request merged**